test/fixtures/schema.sql

```diff
@@ -566,6 +566,12 @@ CREATE TABLE no_pk (
     b character varying
 );
 
+CREATE TABLE no_pk_but_unique (
+    a character varying,
+    b character varying,
+    unique_column character varying DEFAULT NULL UNIQUE
+);
+
```

Comment:
Looks like this table is not being used now, it could be useful for later when the unique Location works. But for now, I think It'd be better to remove it.